### DISCUSSION TITLE
Question regarding notebooks during course

### DISCUSSION THREAD

**Discussion by wolfidan** [openradar/erad2022#61] [General]
Hi,

I have a question regarding how the notebooks are shown during the course. How did you typically do it? Did you do a guided  and commented demonstration of the notebooks in class or did you let the students do it more on their own as an exercise and answer their questions if they have some?

Thanks in advance,
Daniel

**Comment by kmuehlbauer** [comment]
In the preceding years at ERAD it has been something like a hands-on approach. The instructor is moving through the notebook explaining things while the attendees move along in their respective notebooks. They surely can move faster or dwell at some cell to test something. For the wradlib notebooks we give some advise what the attendees could try (eg. "_try to change that parameter and observe the differences_" or "_plot that other radar moment_"). We also have several blocks which we could skip and leave the attendees for later examination (if we run into time problems).

45 Minutes is not that much to have a full featured teaching unit with attached exercise, so the hands-on approach seems a good compromise.
